### DISCUSSION TITLE
Welcome to Ziffers Discussions!

### DISCUSSION THREAD

**Discussion by amiika** [amiika/ziffers#50] [Announcements]
## 👋 Welcome!
  Ziffers Discussions is a place for talking about numbered notation and algorithmic composition:
  * Ask questions you’re wondering about Ziffers
  * Share ideas

  This thread is a test for Discussions feature. Comment away and feel free to test new features :)

**Comment by Bubobubobubobubo** [comment]
Hi @amiika! Nice to see that you are still working on Ziffers :)

**Reply by amiika** [reply]
Nice to see you here too! Took a long time to figure things out, refactor the old parser and document the new syntax. Still at it really ... but it's mostly finished now. Yet there's still some rough edges and many unexplored possibilities :)
